src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3162,20 +3139,54 @@ private TypeSymbolWithAnnotations ApplyConversion(BoundExpression operand, Conve
                     break;
             }
 
-            return TypeSymbolWithAnnotations.Create(targetType, isNullableIfReferenceType);
+            var resultType = TypeSymbolWithAnnotations.Create(targetType, isNullableIfReferenceType);
+
+            // Need to report all warnings that apply since the warnings can be suppressed individually.
```

Comment:
There are still four places that report WRN_NullabilityMismatchInAssignment, two places that call  ReportAssignmentWarnings and a few places that call ReportArgumentWarnings. Could/should those places use ApplyConversion instead?

Reply:
Added a comment for now.

---
In reply to: [211801165](https://github.com/dotnet/roslyn/pull/29429#discussion_r211801165) [](ancestors = 211801165)